torchdynamo/variables/builtin.py

```diff
@@ -94,6 +96,7 @@ def can_constant_fold_through(self):
     @functools.lru_cache(None)
     def _fx_graph_functions():
         fns = {
+            int,
```

Comment:
Perhaps we should be more selective about admitting `int()` into the FX graph.  If someone calls `int(Tensor)` we should likely break the graph on it.  So maybe allow int on a SeedVariable/DynamicShapeVariable but not a general tensor?

Reply:
I agree. I don't like `int()` in the graph. Yes, I think allowing `int` on SeedVariable/DynamicShapeVariable is acceptable.  Will make the changes.